enterprise/api/v1/views/enterprise_group.py

```diff
@@ -117,11 +124,12 @@ def get_learners(self, *args, **kwargs):
             }
 
         """
-
+        q = self.request.query_params.get('q')
```

Comment:
We should really do some validation of these query params before sending them off to the ORM. You could make a serializer class that you pass `self.request.query_params` into and call `is_valid()` on.